modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/components/page-accessibility-tool/components/PanelNavigator.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import ClayIcon from '@clayui/icon';
+import ClayLayout from '@clayui/layout';
+import ClayLink from '@clayui/link';
+import ClayList from '@clayui/list';
+import React from 'react';
+
+function PanelNavigator({helpUrl, impact, onBack, title}) {
+	return (
+		<div className="sidebar-header">
+			<ClayButton
+				className="autofit-row sidebar-section"
+				displayType="unstyled"
+				onClick={onBack}
+				role="tab"
+			>
+				<ClayLayout.ContentCol className="align-self-center mr-2">
+					<ClayIcon symbol="angle-left" />
+				</ClayLayout.ContentCol>
+				<ClayLayout.ContentCol expand>
+					<ClayList.ItemTitle>{title}</ClayList.ItemTitle>
+					<ClayList.ItemText className="text-secondary">
+						{`${impact} - `}
+						<ClayLink
+							className="text-primary"
+							displayType="unstyled"
+							href={helpUrl}
+						>
+							WCAG
```

Comment:
By definition, when I saw this markup not variating around Figma mockups, I search about it and I found the meaning is "Web Content Accessibility Guidelines" which can be understood as a general term for the link to the recommendation docs. I would add @dsanz and @victorvalle to discuss it ;)

Reply:
If our goal is to see the recommendation categories here I would consider to use Labels for it but on our current API We don't have links to separated categories docs

Reply:
Yes, I think Figma is just an example of the information that we are going to use, the `helpUrl` is actually to be used elsewhere, this url is like solving the rule and not which guideline this rule belongs to, because we can have rules which are from `wcag131` and not `wcag2a`. This is also configurable by the checker, so it's weird that it's hardcoded here.

Reply:
Just friendly pinging @dsanz and @victorvalle to see if We could proceed with just this code or would We improve something

We had some conversations on [Slack too](https://liferay.slack.com/archives/C3JBR21HA/p1622650677114700) :p